### PR TITLE
Reorganize modules into one package

### PR DESCRIPTION
**Pull request by merwok**
Trying out idea from @ygingras 

Moves:
- app → `app.core`
- index → `app.__init__` (also moved the Dask app instance there from app.py)
- apps → `app.languages`

I tested that local dev and gunicorn both work.

⚠ do not merge yet

- [x] merge #14 first, then edit target branch of this PR
- [x] discuss with Jeremy

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
 from dash.dependencies import Input, Output
 
-from app import app, server
-from apps import app_en, app_fr, app_zh, app_es
+from .languages import app_en, app_fr, app_zh, app_es
+
+
+app = dash.Dash(__name__, meta_tags=[
+    {'name': 'viewport', 'content': 'width=device-width'},
+    {'name': 'description', 'content': 'COVID-19 Montreal Dashboard / Tableau de bord COVID-19 Montréal'},
```
Comment (merwok) [inline app/__init__.py:11]: I fixed this yesterday, didn’t seem worth moving to another PR

**Comment by jeremymoreau** [comment]
Looks fine to me! This is a different structure than that recommended by the Dash docs (https://dash.plotly.com/urls), but it works and looks cleaner, so I don't see any issues. Thanks!
